toonz/sources/toonzfarm/tfarmcontroller/tfarmcontroller.cpp

```diff
@@ -90,14 +97,16 @@ TFilePath getLocalRoot() {
   TFilePath lroot;
 
 #ifdef _WIN32
-  TFilePath name(L"SOFTWARE\\OpenToonz\\OpenToonz\\1.0\\TOONZROOT");
+  QString regpath = QString::fromStdString("SOFTWARE\\OpenToonz\\OpenToonz\\" + std::string(applicationVersion) + "\\FARMROOT");
```

Comment:
The L prefix just means it's a unicode string with 16 bit characters. It's also what modern Windows and TFilePath use internally. Since neither of these strings contain anything but ASCII/ANSI characters (i.e. no Japanese characters for instance) since the components you are building them from are _const char*_ literals without any UTF8 characters, I don't think there is a need to worry about Unicode here.

QString is what Qt uses internally and thus is used for strings that need to be used with the UI. TFilePath uses std::wstring internally. If you supply a QString, it will just convert it to std::wstring anyway. I think not depending on Qt classes where there is no real gain from them is a good idea since it keeps code more portable.

As for the build and version control system, I'm afraid I can't be of much help, sorry.